site/source/_posts/2018/01/17-Announcing-Dojo-2-beta-5/Announcing-Dojo-2-beta-5.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Announcing Dojo 2 beta 5!
+date: 2018-01-17 09:00:00
+authorId: dylan
+featured_image: featured.png
+---
+
+Dojo is a modern, reactive framework authored with ES2017 and TypeScript, and optimized with the needs of enterprise application development in mind.
+
+If you're learning about Dojo 2 for the first time, please read our overview about the [Dojo 2 beta 4 release](https://dojo.io/blog/2017/12/04/Announcing-Dojo-2-beta-4/).
+
+{% asset_img featured.png feature-full %}
+
+<!-- more -->
+
+## Getting Beta 5
+
+Beta 5 is currently tagged as `latest` in npm, so simply updating to the latest version will provide you with the Beta 5 release.  We also have a release tag of `beta5` in npm if you want to focus just on using Beta 5.
+
+This will be the final beta release for Dojo 2! As we make releases of packages in anticipation of Release Candidate 1, we will not publish those packages to `latest` until Release Candidate 1 is available.
+
+## Just getting started
+
+If you want to get started with Dojo 2, the best way is to take a look at our [tutorials](/tutorials/) or even take a look at Dojo 2 in action with our [example applications](https://dojo.github.io/examples/) (source on [GitHub](https://github.com/dojo/examples)).  Good starting tutorials are:
+
+* [Dojo 2 local installation](/tutorials/000_local_installation/)
+* [Your first Dojo 2 application](/tutorials/001_static_content/)
+* [Creating widgets](/tutorials/003_creating_widgets/)
+
+## New in beta 5
+
+Since the beta 4 release, we have made a number of improvements. The biggest changes and additions are mentioned here, with a complete list of changes
+
+### Initial re-architecture of the build system
+
+Prior to beta5, the structure of the build commands was focused around the tooling and not around the actual creation of applications. So we completed a re-architecture and restructuring of the commands to align more to their use. Specifically, `@dojo/cli-build-webpack` is deprecated and replaced with:
+
+* [`@dojo/cli-build-app`](https://github.com/dojo/cli-build-app/) A command that builds an application
+* [`@dojo/cli-build-widget`](https://github.com/dojo/cli-build-widget/) A command that build a widget, including support for web components
+* Other commands may be planned for the future
+
+This refactoring is mostly complete for Beta 5, with a few remaining items to complete for release candidate 1.
+
+Usage of the build system is reflected in updates to the [Dojo 2 tutorials](https://dojo.io/tutorials/).
+
+### Improvements to using Custom Elements/Web Components
+
+TODO: Explain
```

Comment:
* Added a decorator for the custom element descriptor, meaning that there is no requirement for a separate module any more (https://github.com/dojo/widget-core/pull/792)

That's the only custom element change since `beta4`